apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/Edit/DeleteBusinessRulesWarningModal.scss

```diff
@@ -0,0 +1,4 @@
+.usa-modal__footer {
+    //display: flex;
+    flex-direction: row;
+}
```

Comment:
Potential issue: This may have unintended side effects. It would be better to have a more specific selector instead of changing the flex-direction for all `.usa-modal__footer`s.